quiz/backend/users/tests/test_models.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+
+
+@pytest.mark.django_db
+def test_create_user(django_user_model):
+    user = django_user_model.objects.create_user(
+        email="testuser@test.com",
+        username="testUser",
+        password="testPass123",
+    )
+    saved_user = django_user_model.objects.get(email="testuser@test.com")
+
+    assert saved_user.username == "testUser"
+
+
+@pytest.mark.django_db
+def test_create_user_raises_value_error_when_email_is_empty_string(
```

Comment:
Sprawdziłbym czy dodanie dwóch opcji do field'a w modelu nie zrobi tego samego co customowy manager. Masz gotowy test więc powinno to być dość łatwe.

Reply:
Jak wyżej, null i blank w EmailField są False na deafulcie😅, więc jak teraz to nic nie zmienia, to napisanie ich tym samym obawiam się, że nic nie zmieni.